extensions/wikia/Wall/js/Wall.js

```diff
@@ -302,18 +302,16 @@
 		vote: function (e) {
 			e.preventDefault();
 			if (!window.wgUserName) {
-				require(['AuthModal'], function (authModal) {
-					authModal.load({
-						forceLogin: true,
-						url: '/signin?redirect=' + encodeURIComponent(window.location.href),
-						origin: 'wall-and-forum',
-						onAuthSuccess: this.proxy(function () {
-							this.voteBase(e, function () {
-								window.location.reload();
-							});
-						})
-					});
-				}.bind(this));
+				window.wikiaAuthModal.load({
+					forceLogin: true,
+					url: '/signin?redirect=' + encodeURIComponent(window.location.href),
+					origin: 'wall-and-forum',
+					onAuthSuccess: this.proxy(function () {
+						this.voteBase(e, function () {
+							window.location.reload();
+						});
+					}.bind(this))
```

Comment:
Ah, yeah, that's a leftover code which was making me nuts but I didn't really want to rewrite and retest everything. Will do though.